ci_tools/prepare_linux.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
```

Comment:
Should I make it clear a developer should not run it. That's why it's in the ci_tools directory